### PR TITLE
Update Rule “do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule”

### PR DESCRIPTION
**Pull request by chrisschultzssw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md [hunk 1]
```diff
@@ -9,87 +9,82 @@ authors:
...
 
-* Machine name is COBRA
-* Model: ThinkPad X1 Carbon Gen8
-* Operating System: Windows 10 Enterprise installed and activated
+* Machine name: Bob-SurfacePro9
+* Model: Surface Pro 9
+* Operating System: Windows 11 Enterprise  
 
- **Done** - I have followed an SSW Rule: [https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running](/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running) and this included: 
+I have followed an SSW Rule: [https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running](/do-you-know-the-quickest-way-to-get-your-windows-10-soe-up-and-running) and this included:
```
Comment (kikibianc) [inline rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md:69]: Change URL to https://rules.ssw.com.au/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running (removing the Windows 10 thing)

**Review by kikibianc — CHANGES_REQUESTED** [review]
Looks good, needs 1 fix to be perfect

File: rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md (hunk 1 above)
Comment (chrisschultzssw) [reply rules/do-you-know-the-quickest-way-to-get-your-windows-soe-up-and-running/rule.md]: Done

**Review by kikibianc — APPROVED** [review]
Good job

**Pull request merged**